ui/src/app/views/cohort-builder/search/wizard-tree-children/wizard-tree-children.component.html

```diff
@@ -0,0 +1,25 @@
+<ng-container [clrLoading]="loading">
+  <clr-tree-node *ngFor="let node of nodes">
+    <div class="row clr-treenode-link">
+      <div class="col-xs-9 text-truncate">
+        <span *ngIf="node.code; then showCode else noCode"></span>
+        <ng-template #showCode><small class="font-weight-bold">{{ node.code }}</small>-<small class="text-muted">{{ node.name }}</small></ng-template>
+        <ng-template #noCode><small class="font-weight-bold">{{ node.name }}</small></ng-template>
+      </div>
+      <div class="col-xs-3 text-right">
+        <span *ngIf="node.count === 0; then noCount else count"></span>
+        <ng-template #count><span class="badge badge-light-blue">~{{ node.count }}&nbsp;&nbsp;</span></ng-template>
```

Comment:
I'm wondering if I can propose an alternative to `&nbsp;` here, but I can't get the tree to load at the moment.